Yoast/wordpress-seo
# Fix padding and margin in the container on the metabox

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Open a post-edit page and see the metabox padding being tighter

## Quality assurance


Fixes #
